bedrock/firefox/templates/firefox/new/scene1.html

```diff
@@ -148,12 +152,36 @@ <h4 class="heading-linux">{{_('Download Firefox <span>for Linux</span>') }}</h4>
     </section>
   </div>
 </div>
+
+{% if l10n_has_tag('fxnew_account_032019') %}
+<div id="firefox-account">
+  <h4 class="fxa-title">{{ _('You’ve already got the Firefox browser. Now get everything else Firefox.') }}</h4>
+  <p class="fxa-intro">{{ _('Sync bookmarks, send tabs, save to Pocket and more with a Firefox Account.') }}</p>
```

Comment:
Other than the intro line here, there's little info on what the benefits of an account can be. Could it be worth adding a link to /firefox/accounts/ for the curious types?